vendor/bat-native-ads/src/bat/ads/internal/user_activity/user_activity_util.h

```diff
@@ -6,12 +6,22 @@
 #ifndef BRAVE_VENDOR_BAT_NATIVE_ADS_SRC_BAT_ADS_INTERNAL_USER_ACTIVITY_USER_ACTIVITY_UTIL_H_
 #define BRAVE_VENDOR_BAT_NATIVE_ADS_SRC_BAT_ADS_INTERNAL_USER_ACTIVITY_USER_ACTIVITY_UTIL_H_
 
+#include <cstdint>
 #include <string>
 
+#include "bat/ads/internal/user_activity/user_activity_event_info_aliases.h"
+#include "bat/ads/internal/user_activity/user_activity_event_types.h"
 #include "bat/ads/internal/user_activity/user_activity_trigger_info_aliases.h"
 
 namespace ads {
 
+int GetNumberOfTabsOpened(const UserActivityEventList& events);
```

Comment:
Decouple to user_activity_tab_util

Reply:
as discussed this will be deleted in the next PR, ok to leave then?

Reply:
Ah yes. Thanks